docs/docs/launch-manual/sections/core-hacking/sections/tools-of-the-trade.md

```diff
@@ -0,0 +1,41 @@
+## Tools of the Trade
+
+To begin, there are a few things we'll assume you know, at least to some degree.
+Since all of Pulsar is implemented using web technologies, we have to assume you
+know web technologies such as JavaScript and CSS. Specifically, we'll be using
+Less, which is a preprocessor for CSS.
+
+<!--This whole section needs to be reworked once decaff properly starts on the core-->
+
+While much of Pulsar has been converted to JavaScript, a lot of older code has
```

Comment:
```suggestion
While much of Pulsar has been converted to JavaScript, a lot of older code is
```

Reply:
Oof, that was clearly me changing my mind 5 times :P